tsconfig.json

```diff
@@ -1,27 +1,108 @@
 {
   "compilerOptions": {
-    // Enable latest features
-    "lib": ["ESNext", "DOM"],
-    "target": "ESNext",
-    "module": "ESNext",
-    "moduleDetection": "force",
-    "jsx": "react-jsx",
-    "allowJs": true,
-
-    // Bundler mode
-    "moduleResolution": "bundler",
-    "allowImportingTsExtensions": true,
-    "verbatimModuleSyntax": true,
-    "noEmit": true,
-
-    // Best practices
-    "strict": true,
-    "skipLibCheck": true,
-    "noFallthroughCasesInSwitch": true,
-
-    // Some stricter flags (disabled by default)
-    "noUnusedLocals": false,
-    "noUnusedParameters": false,
-    "noPropertyAccessFromIndexSignature": false
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
Do not leave helper comments generated by a tool in production code. All of the configs and their functionality is available in Typescript official docs.  

In addition, this default `tsconfig` is not a good one for our project. Please add a new issue for updating it.

Reply:
done 🍀